sky/templates/kubernetes-ray.yml.j2

```diff
@@ -426,6 +426,8 @@ available_node_types:
               smarter-devices/fuse: "1"
               {% endif %}
             limits:
+              cpu: {{cpus}}
+              memory: {{memory}}G
```

Comment:
We do not set limits because we want SkyPilot pods to use any idle CPU cycles instead of being throttled to a specific limit. https://home.robusta.dev/blog/stop-using-cpu-limits 

```suggestion
```

Reply:
ok, let me remove it, maybe we should add this notice to docs somewhere.